public/css/site.css

```diff
@@ -380,3 +380,14 @@ strong {
   font-weight: bold;
   font-size: 1.5rem;
 }
+
```

Comment:
where is it used?

Reply:
This class is used to visually hide the screen reader descriptions of the diagrams, so they can be listened to but not seen.